bin/oneoff/backfill_data/channel_tokens_storage_app_id.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+# This script populates the storage_apps_id column of the channel_tokens table.
+
+require_relative '../../../dashboard/config/environment'
+require_relative '../../../shared/middleware/helpers/storage_id'
+
+ChannelToken.find_each do |channel_token|
+  storage_app_id, _channel_id = storage_decrypt_channel_id(channel_token.channel)
+  channel_token.storage_app_id = storage_app_id
+  channel_token.save!
+end
```

Comment:
Not sure what the easiest / least impactful route would be. Doing it all at once would probably cause some user-bad and risk missing a place. Creating the new table, writing to both places, switching the reads, then removing the old table would be less risky though far more work.

It might be possible to do some ruby-foo to alias the table name (as a ruby symbol) in our codebase, mitigating the risk of missing a place for the all at once migration.